p2p/transport/quicreuse/config.go

```diff
@@ -0,0 +1,22 @@
+package quicreuse
+
+import (
+	"net"
+	"time"
+
+	"github.com/lucas-clemente/quic-go"
+)
+
+var quicConfig = &quic.Config{
+	MaxIncomingStreams:         256,
+	MaxIncomingUniStreams:      5,              // allow some unidirectional streams, in case we speak WebTransport
+	MaxStreamReceiveWindow:     10 * (1 << 20), // 10 MB
+	MaxConnectionReceiveWindow: 15 * (1 << 20), // 15 MB
+	RequireAddressValidation: func(net.Addr) bool {
+		// TODO(#1535): require source address validation when under load
+		return false
+	},
+	KeepAlivePeriod: 15 * time.Second,
+	Versions:        []quic.VersionNumber{quic.VersionDraft29, quic.Version1},
+	EnableDatagrams: true,
```

Comment:
Does it make sense to enable this if we aren't using it?

Reply:
Datagram support is a requirement for WebTransport. Chrome won't let us connect if it's not enabled.